### PR TITLE
Network connectivity fixes

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: ethcore/res/ethereum/frontier.json [hunk 1]
```diff
@@ -170,9 +170,6 @@
 		"enode://cadc6e573b6bc2a9128f2f635ac0db3353e360b56deef239e9be7e7fce039502e0ec670b595f6288c0d2116812516ad6b6ff8d5728ff45eba176989e40dead1e@37.128.191.230:30303",
 		"enode://595a9a06f8b9bc9835c8723b6a82105aea5d55c66b029b6d44f229d6d135ac3ecdd3e9309360a961ea39d7bee7bac5d03564077a4e08823acc723370aace65ec@46.20.235.22:30303",
 		"enode://029178d6d6f9f8026fc0bc17d5d1401aac76ec9d86633bba2320b5eed7b312980c0a210b74b20c4f9a8b0b2bf884b111fa9ea5c5f916bb9bbc0e0c8640a0f56c@216.158.85.185:30303",
-		"enode://84f5d5957b4880a8b0545e32e05472318898ad9fc8ebe1d56c90c12334a98e12351eccfdf3a2bf72432ac38b57e9d348400d17caa083879ade3822390f89773f@10.1.52.78:30303",
```
Comment (arkpar) [inline ethcore/res/ethereum/frontier.json]: These have local addresses. Not sure how they ended up here.

**Comment by rphmeier** [comment]
What is this fixing (other than broken bootnodes)? I see some more tracing and a variable rename that looks pretty uncontroversial.

File: util/network/src/session.rs [hunk 2]
```diff
@@ -435,16 +435,16 @@ impl Session {
...
 					ProtocolState::Connected => {
-						trace!(target: "network", "Packet {} mapped to {:?}:{}, i={}, capabilities={:?}", packet_id, protocol, pid, i, self.info.capabilities);
-						Ok(SessionData::Packet { data: packet.data, protocol: protocol, packet_id: pid } )
+						trace!(target: "network", "Packet {} mapped to {:?}:{}, i={}, capabilities={:?}", packet_id, protocol, protocol_packet_id, i, self.info.capabilities);
+						Ok(SessionData::Packet { data: packet.data, protocol: protocol, packet_id: protocol_packet_id } )
 					}
 					ProtocolState::Pending(ref mut pending) => {
 						trace!(target: "network", "Packet {} deferred until protocol connection event completion", packet_id);
-						pending.push((packet.data, packet_id));
+						pending.push((packet.data, protocol_packet_id));
```
Comment (arkpar) [inline util/network/src/session.rs]: Wrong packet id was used here.

Comment (rphmeier) [inline util/network/src/session.rs]: ah, I see. sorry about that

**Pull request merged**